app/controllers/articles_controller.rb

```diff
@@ -240,29 +232,13 @@ def set_article
     Honeycomb.add_field("article_id", @article.id)
   end
 
-  def article_params
-    params[:article][:published] = true if params[:submit_button] == "PUBLISH"
-    modified_params = policy(Article).permitted_attributes
-    modified_params << :user_id if org_admin_user_change_privilege
-    modified_params << :comment_template if current_user.has_role?(:admin)
-    params.require(:article).permit(modified_params)
-  end
-
   # TODO: refactor all of this update logic into the Articles::Updater possibly,
```

Comment:
I still kept some logic here, so kept the comment. There is room for future improvements.